lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -38,13 +33,12 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
       viewportFraction: 0.8,
     )..addListener(() {
         var nextIndex = _pageController.page.round();
-        setState(() {});
-        if (nextIndex != _currentIndex) {
-          _currentIndex = nextIndex;
-          classificationBlocs[_currentIndex].add(TeamClassificationLoadEvent(_teams[_currentIndex]));
-        }
+        setState(() {
```

Comment:
Using 
```
setState(() {});
_currentIndex = ...
```
Or 
```
setState(() {
  _currentIndex = ...
});
```
has the same effect. In fact the `setState` method is just to inform that the widget is dirty and need a rebuild.

When you have a condition to set the widget dirty, you usually do:
```
if (nextIndex != _currentIndex) {
  setState(() { _currentIndex = nextIndex; });
}
```
to rebuild the widget **only** when the `_currentIndex` change !  

But... we have also a manual "animation" (see https://github.com/eric-taix/volley34-mobile/pull/45/files#diff-036760149c06145d95a0df7378c32997R83) which is related to the "distance" between the card and the index (see https://github.com/eric-taix/volley34-mobile/pull/45/files#diff-67aeca6c5edfbe0de0bdb6aaf6cfa864R75): the card zoom in/out according to its position. So we must rebuild the widget when the `PageView` scrolls: this is the purpose of this listener.